lightning/src/ln/functional_test_utils.rs

```diff
@@ -469,33 +469,46 @@ pub fn create_chan_between_nodes_with_value<'a, 'b, 'c, 'd>(node_a: &'a Node<'b,
 	(announcement, as_update, bs_update, channel_id, tx)
 }
 
+/// Gets an RAA and CS which were sent in response to a commitment update
+///
+/// Should only be used directly when the `$node` is not actually a [`Node`].
+macro_rules! do_get_revoke_commit_msgs {
```

Comment:
Does this commit still make sense? It's now a macro that calls a function that calls a macro. Is the LoC reduced documented in the commit message still accurate?